app/controllers/clubs_controller.rb

```diff
@@ -61,11 +61,10 @@ def update
     respond_to do |format|
       if @club.update(club_params)
         format.html { redirect_to edit_club_url(@club), notice: "Club was successfully updated." }
-        format.json { render :show, status: :ok, location: @club }
+        format.turbo_stream { head :ok }
       else
-        flash.now[:alert] = @club.errors.full_messages.join("<br/>")
         format.html { render :edit, status: :unprocessable_entity }
-        format.json { render json: @club.errors, status: :unprocessable_entity }
+        format.turbo_stream { render turbo_stream: turbo_stream.replace("content", partial: "form", locals: { club: @club }) }
```

Comment:
You dont need to delete format.json. You can just add format.turbo_stream no?

Reply:
Right, fixed.